TS06-Plotting-Facets-And-Group-By-Time-In-R.Rmd

```diff
@@ -61,14 +61,13 @@ preferably RStudio, loaded on your computer.
 ****
 
 {% include/_greyBox-wd-rscript.html %}
-{% include/tutorialSeries/_series_dc-tabular-time-series.html %}
 
 ****
 
 ### Recommended Tutorials
-
-This lessons uses both `dplyr` and `ggplot2`. We recommend the following tutorials
-before working through this one, if you are new to either of the above `R` packages.
+This lessons uses both `dplyr` and `ggplot2`. We recommend the following
```

Comment:
looks like we're still back and forth on "lesson" vs tutorial. Shall we stick with tutorial throughout?